### ISSUE TITLE
consider adding support for `numexpr`

### ISSUE THREAD

**Issue by CagtayFabry** [BAMWelDX/weldx#858]
[numexpr](https://github.com/pydata/numexpr) could be an interesting faster alternative to `sympy` in simple cases (but not sure if `xarray` inputs are supported)

**Comment by marscher** [comment]
There is a proposal to add eval to XArray (only Datasets though), but is seems stalled https://github.com/pydata/xarray/pull/7163

**Comment by marscher** [comment]
And even it would be added, we would first need to come up with a proper solution for https://github.com/BAMWelDX/weldx/issues/810

**Comment by CagtayFabry** [comment]
> And even it would be added, we would first need to come up with a proper solution for #810

yes we should certainly look into restoring xarray compatibility before anything else
